### PR TITLE
fixed head pic & badges

### LINKED ISSUE #6: [README] Need to fix head-pic & slack badge
**Description**

<a href="https://github.com/layer5io/layer5-istio-adapter/blob/master/README.md">README.md</a> is the first file one should read when starting a new project. For attractive look, we added some badges also.
But, there's a head picture & Slack badge failing, need to fix that. 
 
 **Note:**
 Head picture (at top) recognize to this repo. only.
 
### Example Screenshot

**Current Behavior**

image

**Desired Behavior**
image

**Environment:**
 - OS: any [e.g. Ubuntu]
 - Browser: any [e.g. Chrome, Safari]
 - Version: any [e.g. 22]
 - Device: any [e.g. laptop, iPhone 8]

### PR DESCRIPTION
**Pull request by singh08prashant**
fixes #6 
Signed-off-by: Prashant Singh <singh08prashant@gmail.com>

### CODE REVIEW

**Comment by amino19** [comment]
@singh08prashant : Can you pl rework on it? [Only change the slack badge i.e, failing]
For place it as `center`, Pl take the reference from <a href="https://github.com/service-mesh-performance/service-mesh-performance/pull/196/commits/4ce829327186b4b9a56002fd49a554a5266437ed">here</a>.
Else Good :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+
+<p align="center">
+<a href="https://github.com/layer5io/layer5" alt="GitHub contributors">
+<img src="https://img.shields.io/github/contributors/layer5io/layer5.svg" /></a>
+<a href="https://github.com/layer5io/layer5" alt="License">
+<img src="https://img.shields.io/github/license/layer5io/layer5.svg" /></a>
+<a href="https://github.com/layer5io/layer5" alt="GitHub issues by-label">
+<img src="https://img.shields.io/github/issues/layer5io/layer5/help%20wanted.svg?color=%23DDDD00" /></a>
+<a href="https://github.com/layer5io/layer5" alt="Slack">
+<img src="https://img.shields.io/badge/Slack-@layer5.svg?logo=slack" /></a>
```
Comment (aminoxix) [inline README.md]: @singh08prashant : Can you pl rework on it? [Only change the slack badge i.e, failing]
For place it as `center`, Pl take the reference from <a href="https://github.com/service-mesh-performance/service-mesh-performance/pull/196/commits/4ce829327186b4b9a56002fd49a554a5266437ed">here</a>.
Else Good! :slightly_smiling_face:

**Review by aminoxix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (singh08prashant) [reply README.md]: Thank you! I'll make changes accordingly.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/src/assets/images/layer5/layer5-tagline/png/layer5-tag-community-white-bg.png" width="45%" /></p>
+
+<div align="center">
+
 ![GitHub contributors](https://img.shields.io/github/contributors/layer5io/layer5.svg)
-![GitHub](https://img.shields.io/github/license/layer5io/layer5.svg) 
+![GitHub](https://img.shields.io/github/license/layer5io/layer5.svg)
 ![GitHub issues by-label](https://img.shields.io/github/issues/layer5io/layer5/help%20wanted.svg?color=%23DDDD00)
-[![Slack](http://slack.layer5.io/badge.svg)](http://slack.layer5.io)
+[![Slack](https://img.shields.io/badge/Slack-@layer5.svg?logo=slack)](http://slack.layer5.io)
```
Comment (amino19) [inline README.md:9]: @singh08prashant : LGTM :partying_face:

File: README.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-community-white-bg.png" width="45%" /></p>
-  
+  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/src/assets/images/layer5/layer5-tagline/png/layer5-tag-community-white-bg.png" width="45%" /></p>
```
Comment (aminoxix) [inline README.md:2]: @leecalcote : Here using, Layer5 logo only. Take this to your end :slightly_smiling_face:

**Review by aminoxix — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aminoxix) [inline README.md:9]: @singh08prashant : Thanks for rework, LGTM! :partying_face:

**Review by aminoxix — APPROVED** [review]

**Comment by leecalcote** [comment]
Thanks, @singh08prashant 👍 and @amino19 😄

**Pull request merged**